ExampleCode/C++/Apps/Makefile

```diff
@@ -0,0 +1,111 @@
+# Makefile for the iSENSE projects involving C++
+# Designed to quickly compile the C++ code (only if files are changed)
+CC = g++
+
+# NOTES: -lcurl is required. std=c++11 is also needed for to_string. -ansci
+CFLAGS = -Wall -Werror -pedantic -std=c++11 -lcurl
+
+# In the future, should try compiling with these flags (change to -ansi)
+# CFLAGS= -Wall -Werror -ansi -pedantic
+Boost= -lboost_unit_test_framework
+
+# Makes all of the C++ projects, appends a ".out" for easy removal in make clean
+all:	baseball_app.out dice_app.out
+
+baseball_app.out:	baseball_app.o API.o
+	$(CC) baseball_app.o API.o -o baseball_app.out $(CFLAGS)
+
+baseball_app.o:	baseball_app.cpp
+	$(CC) -c baseball_app.cpp $(CFLAGS)
+
+dice_app.out:	dice_app.o API.o
+	$(CC) dice_app.o API.o -o dice_app.out $(CFLAGS)
+
+dice_app.o:	dice_app.cpp
+	$(CC) -c dice_app.cpp $(CFLAGS)
+
+API.o:	API.cpp include/API.h include/memfile.h include/picojson.h
+	$(CC) -c API.cpp $(CFLAGS)
+
+
+# No longer using:
+# append_email_byID.out append_key_byID.out
+#
+# # Contributor key test
+# key.out:	POST_key.o API.o
+# 	$(CC) POST_key.o API.o -o key.out $(CFLAGS)
+#
+# POST_key.o:	POST_key.cpp
```

Comment:
I should probably fix the Makefile, I commented some stuff out after I moved stuff around.
I'll probably just change it so "all" calls the tests.cpp, but the other commands can still be called if a user wants to try it (Ex: make key.out if they want to try using contributor keys).